rust/kcl-lib/src/parsing/token/tokeniser.rs

```diff
@@ -51,14 +51,6 @@ lazy_static! {
         set.insert("struct", TokenType::Keyword);
         set.insert("object", TokenType::Keyword);
 
-        set.insert("string", TokenType::Type);
-        set.insert("number", TokenType::Type);
-        set.insert("bool", TokenType::Type);
-        set.insert("Sketch", TokenType::Type);
-        set.insert("SketchSurface", TokenType::Type);
-        set.insert("Solid", TokenType::Type);
-        set.insert("Plane", TokenType::Type);
```

Comment:
Removing from this set means they're no longer reserved. So do we allow users to define variables with these names now?

Reply:
Yep! Although types are in a different name space so there is never a clash